js/app/location.js

```diff
@@ -130,16 +131,14 @@ define([
             },
             draw: (renderer, color) => {
                 return new Promise((resolve) => {
-                    var topLeftCorner = renderer.convertCoordonateToBackground(data.description.topLeftCorner);
-                    var bottomRightCorner = renderer.convertCoordonateToBackground(data.description.bottomRightCorner);
+                    var topLeftCorner = renderer.convertCoordonateToBackground(data.description.topLeftCorner, true);
```

Comment:
I didnt see the updated methods with the last arguments. https://github.com/The-Brains/ClickAndPointLib/blob/master/js/app/renderer.js#L55-L75

Reply:
It's in this review